Scripts/RemediationScripts/Remediate-InvalidAADObjectRoleAssignments.ps1

```diff
@@ -300,23 +332,27 @@ function Remove-AzTSInvalidAADAccounts
         return;
     }
 
-    # Getting count of invalid account
+    # Getting count of deprecated/invalid accounts
     $invalidAADObjectRoleAssignmentsCount = ($invalidAADObjectRoleAssignments | Measure-Object).Count
     $invalidClassicRolesCount = ($invalidClassicRoles | Measure-Object).Count
+    $ascDeprecatedRolesCount = ($ascDeprecatedRoleAssignmentList | Measure-Object).Count
 
-    if(($invalidAADObjectRoleAssignmentsCount -eq 0) -and ($invalidClassicRolesCount -eq 0))
+    
+
+    if(($invalidAADObjectRoleAssignmentsCount -eq 0) -and ($invalidClassicRolesCount -eq 0) -and ($ascDeprecatedRolesCount -eq 0))
     {
-        Write-Host "No invalid accounts found for the subscription [$($SubscriptionId)]. Exiting the process."
+        Write-Host "No invalid accounts/deprecated accounts found for the subscription [$($SubscriptionId)]. Exiting the process."
```

Comment:
Nit: This can be `invalid/deprecated` accounts.

Reply:
Added 'invalid/deprecated accounts.'